catalog/PartitionSchemeHeader.hpp

```diff
@@ -48,6 +50,13 @@ class Type;
  **/
 class PartitionSchemeHeader {
  public:
+  // A vector for partitioning catalog attribute .
+  typedef std::vector<attribute_id> PartitionAttributeIds;
+
+  // A boundary for multi-attribute partitions.
+  // PartitionValues.size() should be equal to PartitionAttributeIds.size().
+  typedef std::vector<TypedValue> PartitionValues;
```

Comment:
Sorry, the comment is not that clear. This represents the corresponding values for `PartitionAttributeIds`, and is also used to store the partition range boundaries for a range-based partition.